packages/cli/src/Queue.ts

```diff
@@ -30,7 +30,29 @@ export class Queue {
 		this.activeExecutions = ActiveExecutions.getInstance();
 
 		const prefix = config.getEnv('queue.bull.prefix');
-		const redisOptions = config.getEnv('queue.bull.redis');
+		const redisHost = config.getEnv('queue.bull.redis.host');
+		const redisUsername = config.getEnv('queue.bull.redis.username');
+		const redisPassword = config.getEnv('queue.bull.redis.password');
+		const redisPort = config.getEnv('queue.bull.redis.port');
+		const redisDB = config.getEnv('queue.bull.redis.db');
+		// retro-compatibility with redis < 6
+		// prepare new redis options setting in order to define only set values
+		const redisOptions: IDataObject = {};
```

Comment:
you could use `import type { RedisOptions } from 'ioredis';` and
```suggestion
		const redisOptions: RedisOptions = {};
```

Reply:
Right, commit updated